### DISCUSSION TITLE
Escape pod extension

### DISCUSSION THREAD

**Discussion by SlickSidious** [cyb3rdog/Cyb3rVector#11] [Q&A]
I have the “escape pod extension not connected” Message in code lab. I see under “How do I configure the escape pod to use these?” It’s says “at bare minimum, you’ll need to set the following variables in /etc/escape-pod.conf” then the 3 variables. How/where do I access /etc/escape-pod.conf to change these variables on a PC?

**Comment by SlickSidious** [comment]
I found the Escapepod extension proxy settings but I’ve never created a user name or password for escape pod nor do I know where to do this. And also is the Host or IP supposed to be escapepod.local?

**Reply by cyb3rdog** [reply]
Hi, just deploy the extension to your escape pod with deploy button. It will setup everything by itself and you won't need to tickle any configuration files at all. Just as the text there says, in case you have never logged in to your escapepod with ssh, the default username and password will both be 'ubuntu', and also both host name and ip should work for connecting the app to your escapepod. Basically, if you never tweak arround anything on your escapepod, you shouln't need to change anything in that screen, just press the button. Only condition for this work is that vector, escapepod and the pc with Cyb3rVector will all be on the same network...

![image](https://user-images.githubusercontent.com/12493945/147384016-2e881ef3-0e9c-4dc0-80d8-e345b4a9317f.png)

Hope this will help you...

**Reply by SlickSidious** [reply]
Thanks for replying!
So I have my Pi 4 plugged directly into my router, my laptop is connected to my router, and Cyb3rvector is connected vector. Ive left the following settings default 
Host or IP : escapepod.local
User Name : ubuntu 
Password : ubuntu 

the message I keep getting is “No such host is known” when I click “deploy escapepod extension”. Am I missing something?

**Reply by moribundant** [reply]
You may want to ssh into your EP just to make sure the connection is there. The instructions for doing this are on the DDL EP website.

I am assuming that the EP is working with your Vector. Just 'ssh escapepod.local -l ubuntu' from a terminal. If this gives an error, it means that _escapepod.local_ is not mapping to the EP address--this gets done when you onboard the EP. If you know the address of the EP you can just go direct with it, but I would get the _escapepod.local_ mapping straightened out.

The first time you ssh the password is ubuntu--I suggest changing it. Once you are 'into' the EP, then Cyb3rvector should have no issues connecting.

**Reply by gtchtc07** [reply]
What am I missing I try to link to the escape pod extension and keep getting a host error ? Please advise
.....
Everything else on escape pod working fine except the cyb3r code lab extension
